### PR TITLE
CONFIG SET returns OK status

### PR DESCRIPTION
**Pull request by sazzad16**
So we should make `configSet` to match with all other commands that return `OK` status.

### CODE REVIEW

**Review by dengliming — DISMISSED** [review]

File: src/main/java/redis/clients/jedis/BinaryJedis.java [hunk 1]
```diff
@@ -3694,10 +3694,10 @@ public String configRewrite() {
    * @return Status code reply
    */
   @Override
-  public byte[] configSet(final byte[] parameter, final byte[] value) {
+  public String configSet(final byte[] parameter, final byte[] value) {
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: Shouldn't we deprecate the original method first?

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: @gkorland We are not removing the method. So I don't support deprecation. My idea is to add a warning message in jdoc. We can do that after we agree on this PR.

Comment (gkorland) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: I know we are not but we change the signature which will break both compile and runtime

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: Are you suggesting to deprecate without an alternate?

Comment (gkorland) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: no, I suggest we add an new method and deprecate the old one

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3697]: @gkorland check #2525

File: src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java [hunk 2]
```diff
@@ -131,7 +131,8 @@ public void configGetSetBinary() {
     List<byte[]> info = jedis.configGet(maxmemory);
     assertArrayEquals(maxmemory, info.get(0));
     byte[] memory = info.get(1);
-    assertArrayEquals("OK".getBytes(), jedis.configSet(maxmemory, memory));
+    assertEquals("OK", jedis.configSet(maxmemory, memory));
+    assertEquals("OK", jedis.configSet(maxmemory, memory));
```
Comment (gkorland) [inline src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java:135]:
```suggestion
    assertEquals("OK", jedis.configSetBinary(maxmemory, memory));
```
Why do you test the same method twice?

**Review by gkorland — CHANGES_REQUESTED** [review]

File: src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java (hunk 2 above)
Comment (sazzad16) [reply src/test/java/redis/clients/jedis/tests/commands/ControlCommandsTest.java:136]: @gkorland first one should be `200`. fixed.

**Review by dengliming — APPROVED** [review]

**Review by gkorland — APPROVED** [review]

**Pull request merged**